main/ui/src/main/java/org/cryptomator/ui/common/VaultService.java

```diff
@@ -165,6 +164,8 @@ public WaitForTasksTask(Collection<Task<Vault>> tasks) {
 		public LockVaultTask(Vault vault, boolean forced) {
 			this.vault = vault;
 			this.forced = forced;
+
+			setOnFailed(event -> LOG.error("Failed to lock " + vault.getDisplayName(), event.getSource().getException()));
```

Comment:
What is the reason behind this change? Shouldn't be either both (succeeded and failed Tasks) or none of them be moved ?

Reply:
You're right, didn't see this before. Same reasoning as with the UnlockWorkflow: Use of the setter should stay with the caller.

Reply:
The point is (again): The caller could set its own listener, effectively annihilating all these error handlers. This may be ok.

The question is, and I don't think there is an "correct" answer: Is it desired that errors are no longer logged, if the caller overwrites this "default" handler (as its done with the ToggleAutoStartTask)? If not, rather log the error in the Task's `failed()` method.

Reply:
> Is it desired that errors are no longer logged, if the caller overwrites this "default" handler (as its done with the ToggleAutoStartTask)? If not, rather log the error in the Task's failed() method.
That's desired, yes. If I put the logging into the `failed()` method the caller couldn't override the handling anymore and they should be able to do that.